src/embind/embind.js

```diff
@@ -907,6 +907,14 @@ var LibraryEmbind = {
         throwBindingError('function ${humanName} called with ${arguments.length} arguments, expected ${argCount - 2} args!');
       }`;
 
+#if EMBIND_TRACING
```

Comment:
Maybe EMBIND_DEBUG to match all the other `_DEBUG` settings we have.

Also, can you use `dbg` rather than `console.log`?

Reply:
Thanks for the feedback, is `dbg` really an usable function ?

Reply:
`dbg()` is an emscripten-defined function yes.

Reply:
Okay I didn't knew about it because it's only present when `RUNTIME_DEBUG` is True (if I understand correctly)

I have modified the code to activate `RUNTIME_DEBUG`